kong/runloop/balancer.lua

```diff
@@ -1091,20 +1091,22 @@ local function get_balancer_health(upstream_id)
   end
 
   local healthchecker
+  local balancer_status
   local health = "HEALTHCHECKS_OFF"
   if is_upstream_using_healthcheck(upstream) then
     healthchecker = healthcheckers[balancer]
     if not healthchecker then
       return nil, "healthchecker not found"
     end
 
-    local balancer_status = balancer:getStatus()
+    balancer_status = balancer:getStatus()
     health = balancer_status.healthy and "HEALTHY" or "UNHEALTHY"
   end
 
   return {
     health = health,
     id = upstream_id,
+    details = balancer_status,
```

Comment:
On which endpoints would this extra info appear? (A test demonstrating it would be nice. :) )